docs/src/components/previews/CloudImagePreview.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import { useState } from 'react';
+import { ProgressCircle } from '@voussoir/progress';
+import { getDefaultSrcSet } from '../../utils';
+import { CONTENT_MAX_WIDTH_DESKTOP } from '../../constants';
+
+export function CloudImagePreview(props: any) {
+  const src = props.fields.href.value;
+  const alt = props.fields.alt.value;
+  const sizes = props.fields.sizes.value;
+
+  const [isLoading, setIsLoading] = useState(!!src);
+
+  return (
+    <>
+      <p>Href: {src}</p>
+      <p>Alt text: {alt}</p>
+
+      {isLoading && <ProgressCircle aria-label="Loading…" isIndeterminate />}
+      <img
```

Comment:
I was playing around a little, and wondered if the preview also could have the `height` and `width` so you can see it change after you've edited and click "Done"?

Reply:
Yeah, good thinking. I would actually like to programatically add defs a height (if one isn't supplied) but probably also a width as well. `next/image` requires these and they will provide a much better UX. Somehow storing an aspect ratio would probably be sufficient to figure it out.

I can add the height/width indicators, though.